app/src/main/java/tech/pegasys/eth2signer/Eth2SignerApp.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.eth2signer;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import tech.pegasys.eth2signer.commandline.CommandlineParser;
+import tech.pegasys.eth2signer.commandline.Eth2SignerCommand;
+
+import java.io.PrintWriter;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Eth2SignerApp {
+
+  private static final Logger LOG = LogManager.getLogger();
+
+  public static void main(final String... args) {
+    System.out.println("Eth2Signer has started with args" + String.join(",", args));
```

Comment:
Should this be using LOG.info (or debug) instead of System.out??

Reply:
Also, technically we start after parseCommandLine returns, don't we?

Reply:
This is executed prior to parseComandLine - but we probably want to capture this quite early, so we know what's happened.

Reply:
👍